libfwupd/fwupd-client.h

```diff
@@ -78,6 +78,8 @@ gboolean	 fwupd_client_unlock			(FwupdClient	*client,
 							 const gchar	*device_id,
 							 GCancellable	*cancellable,
 							 GError		**error);
+gboolean	fwupd_client_activate_device		(FwupdClient *client, GCancellable *cancellable,
```

Comment:
A little voice in my head is telling me the `_device` suffix isn't required. I also don't know if "activate" is the right noun, maybe "commit" or "switch" might be better? Ideas welcome.

Reply:
If we don't use activate here, we might want to switch it for the argument to fwupdmgr and fwuptdool too.